x-pack/test/functional/apps/index_management/data_streams_tab/data_streams_tab.ts

```diff
@@ -23,15 +23,37 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     before(async () => {
       await log.debug('Creating required data stream');
       try {
-        await dataStreams.createDataStream(
-          TEST_DS_NAME,
-          {
-            '@timestamp': {
-              type: 'date',
+        await es.cluster.putComponentTemplate({
+          name: `${TEST_DS_NAME}_mapping`,
+          template: {
+            settings: { mode: undefined },
+            mappings: {
+              properties: {
+                '@timestamp': {
+                  type: 'date',
+                },
+              },
             },
+            lifecycle: {
+              // @ts-ignore-next-line
```

Comment:
I _think_ the preferred approach is:
```
// @ts-expect-error @elastic/elasticsearch  <<insert comment >>
```

That way, when the types are updated, it's easy to find and remove this line.